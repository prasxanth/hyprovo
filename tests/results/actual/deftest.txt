2021-11-24 17:25:51.806 - INFO - Pass ✓... (test-add-report-combined-results): add pass
2021-11-24 17:25:51.808 - ERROR - Fail ✗... (test-add-report-combined-results): add fail
2021-11-24 17:25:51.809 - DEBUG - (test-add-report-combined-results): Test Summary → Total: 2    Pass: 1    Fail: 1
