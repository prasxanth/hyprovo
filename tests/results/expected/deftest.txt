2021-11-24 17:24:55.395 - INFO - Pass ✓... (test-add-report-combined-results): add pass
2021-11-24 17:24:55.398 - ERROR - Fail ✗... (test-add-report-combined-results): add fail
2021-11-24 17:24:55.398 - DEBUG - (test-add-report-combined-results): Test Summary → Total: 2    Pass: 1    Fail: 1
